backend/controllers/addUserCaseload.ts

```diff
@@ -1,4 +1,6 @@
 import { Request, Response } from 'express'
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { AuditService } = require('../services/auditService')
```

Comment:
Would `import AuditService from '../services/auditService'` work to avoid the eslint disable line?